server/controllers/UserController.js

```diff
@@ -1,10 +1,27 @@
 const db = require("../models/index");
 const User = db.User;
 
+exports.findAll = async (req, res) => {
```

Comment:
Is this an api to find all users? If so what is the use case?

Reply:
Originally I just had that to test the sample API and forgot to remove, I removed it now